core/templates/dev/head/pages/landing/fractions/Fractions.js

```diff
@@ -22,16 +22,24 @@ oppia.controller('Fractions', [
   function($scope, $timeout, $window,
       siteAnalyticsService, UrlInterpolationService) {
     $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+
     $scope.getStaticSubjectImageUrl = function(subjectName) {
       return UrlInterpolationService.getStaticImageUrl('/subjects/' +
         subjectName + '.svg');
     };
+
     $scope.onClickGetStartedButton = function(viewerType) {
       siteAnalyticsService.registerOpenFractionsFromLandingPageEvent(
         viewerType);
       $timeout(function() {
         $window.location = ' /collection/4UgTQUc1tala';
       }, 150);
     };
+
+    $scope.onClickLearnMoreButton = function() {
+      $timeout(function() {
+        $window.location = 'https://www.oppia.org/splash';
```

Comment:
`/splash`, otherwise it'll go to the wrong place when you're testing on the test server.

Reply:
Okay. fixed that